docs/openj9_support.md

```diff
@@ -42,14 +42,13 @@ columns will be removed over time.
+| OpenJ9 release | Release date  | JDK8 (LTS) | JDK11 (LTS) | JDK17 (LTS) | JDK19 | JDK20 |  JDK21 (LTS)  |
+|----------------|---------------|------------|-------------|-------------|-------|-------|---------|
+| 0.37.0         | Mar 2023 (2)  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     |            |          |
+| 0.38.0         | Apr 2023   | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     |            |           |
+| 0.39.0         | Jun 2023 (2)  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>      | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>      |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>          |               |
+| 0.40.0         | Jul 2023   | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>          |              |
+| 0.40.1         | Sep 2023 (1, 3)   |  :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> |  :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> |  :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     |   :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>          |   :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>          |
```

Comment:
There is no 0.40.1 release in Sept. You can add a 0.41.0 release in Oct, which may or may not include support for JDK21.

Reply:
Will this 0.41.0 release be one of those releases that support a new LTS OpenJDK release only or all the listed JDKs including 21?

Reply:
Removing JDK21 for now is fine with me.